fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformServiceImpl.java

```diff
@@ -1399,23 +1426,70 @@ public LoanScheduleData extractData(@NotNull final ResultSet rs) throws SQLExcep
                     totalPaidLate.getAmount(), totalOutstanding.getAmount(), totalCredits.getAmount());
         }
 
-        private BigDecimal processDisbursementData(LoanScheduleType loanScheduleType, Collection<DisbursementData> disbursementData,
-                LocalDate fromDate, LocalDate dueDate, Set<Long> disbursementPeriodIds, BigDecimal disbursementChargeAmount,
-                BigDecimal waivedChargeAmount, List<LoanSchedulePeriodData> periods) {
+        private BigDecimal processDisbursementAndCapitalizedIncomeData(LoanScheduleType loanScheduleType,
```

Comment:
Please make distinction between disbursement and capitalized income! We should not merge them together!
Disbursement amount shall NOT include capitalized income!